call-home/src/bin/callhome/collector/report_models.rs

```diff
@@ -64,33 +66,35 @@ impl VolumeReplicaCounts {
     // Receives a openapi::models::Volumes object and returns number of volumes with specific number
     // of replicas.
     fn new(volumes: &[Volume]) -> Self {
+        let (one_replica, two_replicas, three_replicas, four_replicas, five_or_more_replicas): (
+            u32,
+            u32,
+            u32,
+            u32,
+            u32,
+        ) = volumes.iter().fold(
+            (0, 0, 0, 0, 0),
+            |(one, two, three, four, five_or_more), vol| match vol.spec.num_replicas {
```

Comment:
how does this reduce memory usage?
Though it's probably more probably more efficient time wise

Reply:
Yeah, doesn't reduce any memory usage. Makes things a little faster.